packages/main/src/TimePickerClock.hbs

```diff
@@ -0,0 +1,51 @@
+<div
+	id="{{_id}}"
+	ondragstart="return false;"
+	ondrop="return false;"
+	aria-hidden="true"
+	class="{{classes.clock}}"
+>
+	<div
+		data-label="{{label}}"
+		class="ui5-tp-clock-dial"
+	>
+	</div>
+
+	<div>
+	{{#each _items}}
+		<div class="ui5-tp-clock-item" style="--tpc-degree: {{this.angle}}deg">
```

Comment:
@vladitasev could you check if the following is CSP compliant?
```html
<div class="ui5-tp-clock-item" style="--tpc-degree: {{this.angle}}deg">
```

Reply:
Done.